README.md

```diff
@@ -54,30 +65,33 @@ The plugin is still under development. If you would like to contribute, install
 pip install -e .[dev] --index-url https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple
 ```
 
-
 ### Setting up plugin on your local installation
 Read the [NOMAD plugin documentation](https://nomad-lab.eu/prod/v1/staging/docs/howto/oasis/plugins_install.html) for all details on how to deploy the plugin on your NOMAD instance.
 
 You need to modify the ```src/nomad_simulations/nomad_plugin.yaml``` to define the plugin adding the following content:
+
 ```yaml
-plugin_type: schema
+plugin_type: schemas
 name: schemas/nomad_simulations
 description: |
   This is a collection of NOMAD schemas for simulation data.
 ```
 
 and define the ```nomad.yaml``` configuration file of your NOMAD instance in the root folder with the following content:
+
 ```yaml
 plugins:
-  include: 'schema/nomad_simulations'
+  include:
+    - schemas/nomad_simulations
```

Comment:
Is this listing necessary? It is not appearing in the nomad.yaml like this either

Reply:
Yes, but it gave me issues without it. I'd leave it.